docs/rules-engine/how-to-use/README.md

```diff
@@ -0,0 +1,410 @@
+# Rules engine - Integration
```

Comment:
Isn't this page what we should have in the showcase application ?

Reply:
Well we should probably rework the showcase so that it matches our recommendation and then make reference to the code in the documentation. 
This PR is now getting a bit long, I would rather do the move of the showcase in another one.